src/contrib/persistence/Akka.Persistence.Query.Sql/SqlReadJournal.cs

```diff
@@ -0,0 +1,175 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SqlReadJournal.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Reactive.Streams;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Persistence.Journal;
+using Akka.Streams.Dsl;
+
+namespace Akka.Persistence.Query.Sql
+{
+    public class SqlReadJournal : IReadJournal,
+        IAllPersistenceIdsQuery,
+        ICurrentPersistenceIdsQuery,
+        IEventsByPersistenceIdQuery,
+        ICurrentEventsByPersistenceIdQuery,
+        IEventsByTagQuery,
+        ICurrentEventsByTagQuery
+    {
+        public static string Identifier = "akka.persistence.query.journal.sql";
+
+        /// <summary>
+        /// Returns a default query configuration for akka persistence SQLite-based journals and snapshot stores.
+        /// </summary>
+        /// <returns></returns>
+        public static Config DefaultConfiguration()
+        {
+            return ConfigurationFactory.FromResource<SqlReadJournal>("Akka.Persistence.Query.Sql.reference.conf");
+        }
+
+        private readonly Akka.Serialization.Serialization _serialization;
+        private readonly TimeSpan _refershInterval;
+        private readonly string _writeJournalPluginId;
+        private readonly int _maxBufferSize;
+
+        public SqlReadJournal(ExtendedActorSystem system, Config config)
+        {
+            _serialization = system.Serialization;
+            _refershInterval = config.GetTimeSpan("refresh-interval");
+            _writeJournalPluginId = config.GetString("write-plugin");
+            _maxBufferSize = config.GetInt("max-buffer-size");
+        }
+
+        /// <summary>
+        /// <para>
+        /// <see cref="AllPersistenceIds"/> is used for retrieving all `persistenceIds` of all
+        /// persistent actors.
+        /// </para>
+        /// The returned event stream is unordered and you can expect different order for multiple
+        /// executions of the query.
+        /// <para>
+        /// The stream is not completed when it reaches the end of the currently used `persistenceIds`,
+        /// but it continues to push new `persistenceIds` when new persistent actors are created.
+        /// Corresponding query that is completed when it reaches the end of the currently
+        /// currently used `persistenceIds` is provided by <see cref="CurrentPersistenceIds"/>.
+        /// </para>
+        /// The LevelDB write journal is notifying the query side as soon as new `persistenceIds` are
```

Comment:
You have LevelDB and Sqlite in XML docs. Maybe you should change all string to Sqlite?